.babelrc

```diff
@@ -1,6 +1,26 @@
 {
-  "presets": [
-    "env",
-    "react"
-  ]
+  "env": {
+    // Compatibility Profile. ES5 output and CommonJS module format.
+    "commonjs": {
+      "presets": [
+        ["env", {
+          "useBuiltIns": false
+        }],
+        "react"
+      ]
+    },
+    // Future Profile. ES6 output and module syntax.
+    "es": {
+      "presets": [
+        ["env", {
+          "useBuiltIns": false,
+          "modules": false,
+          "targets": {
+            "node": "6.5"
```

Comment:
Why did you need to fix node version?

Reply:
Yeah Dani, this is a way to make sure Babel transpiles to full ES6. Maybe is not that elegant, but the problem is `babel-preset-env` cannot be targeted to a specific ECMAScript version (e.g.: ES6). They lack a setting to do so unfortunately. Actually, have some open tickets requesting this feature.
Instead, you can target towards a browser version or a node version. This way, the preset will configure Babel by enabling/disabling the required plugins/transformations for that particular version. 

So, in order to make sure we transpile to ES2015, we have to options:
- Manually enable plugins and transformations for ES7, ES8, etc features and disable the corresponding for ES6.
- Or, a simpler aproach, target to Node 6.5 wich was the first version with full ES6 compatibility. 

You can check it with [Babel REPL](https://babeljs.io/repl/).

If you guys know a better way to achieve this, please let me know.

Reply:
If you think guys this could be a problem, I can remove `es` build target, it is not mandatory in NPM packages. Whatever you prefer. This is the only thing pending to confirm, I think the rest is ready.